server/server.js

```diff
@@ -62,7 +75,10 @@ const startServer = () => {
     ]);
 
     app.use(`${basePath}/static`, express.static(buildPath + '/static'));
-    app.use(`${basePath}/asset-manifest.json`, express.static(`${buildPath}/asset-manifest.json`));
+
+    app.get(`${basePath}/asset-manifest.json`, (req, res) => {
+        res.type('json').send(manifestMedFlereFiler(`/rekrutteringsbistand-kandidat/${envPath}`));
```

Comment:
Går det an å lagre det utvidede manifestet i minnet, dvs. kjøre `manifestMedFlereFiler` kun ved oppstart? Nå blir det vel en lese- og skriveoperasjon hver gang noen går inn i appen

Reply:
Hvis vi gjør det ved oppstart hadde det også vært nice med en console.log som informerer om operasjonen, typ `Utvidet asset-manifest.json med referanse til miljøvariabler (<path>)`

Reply:
Du kan også bruke `basePath` til å bygge opp URL-en her – men du kunne også flytta det inn i `manifestMedFlereFiler`-funksjonen, for alle filer skal vel ha med basePathen? Da hadde du også sluppet å parse for å hente ut navnet i `manifestMedFlereFiler`